### PR TITLE
Release 9.2.1 (#897)

### PR DESCRIPTION
**Pull request by lojzatran**
#### Summary
Prepare new release

### CODE REVIEW

File: docs/RELEASE_NOTES.md [hunk 1]
```diff
@@ -27,6 +27,18 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
 
+### 9.2.1 - Feb 02, 2023
```
Comment (praveenkumarct) [inline docs/RELEASE_NOTES.md:30]: Do you want to change the date?

**Review by praveenkumarct — APPROVED** [review]

File: docs/RELEASE_NOTES.md (hunk 1 above)
Comment (lojzatran) [reply docs/RELEASE_NOTES.md:30]: It doesn't matter, I would leave it as it is. The changes for this release is done on 2.2.2023, so it's correct.

Comment (praveenkumarct) [reply docs/RELEASE_NOTES.md:30]: But you will create a tag today. When user checks for release notes, he might gets confused.

Comment (lojzatran) [reply docs/RELEASE_NOTES.md]: https://github.com/commercetools/commercetools-sync-java/pull/905/commits/8265aa7777489d12176f617e6c8b35b23920a156

**Pull request merged**